src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java

```diff
@@ -86,6 +89,7 @@
      */
     private static final String TAG_COLONY_ID      = "colony";
     private static final String TAG_CITIZEN        = "citizen";
+    private static final String TAG_INVENTORY      = "inventory";
     private static final String TAG_HELD_ITEM_SLOT = "HeldItemSlot";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused "TAG_HELD_ITEM_SLOT" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1068)